<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="MainPage">
       <property name="styleSheet">
        <string notr="true">MyLabel:hover
{
	background-color: rgb(159, 135, 255);
	font: 9pt &quot;华文琥珀&quot;;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame_1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">MyLabel：hover
{
background-color: rgba(206, 206, 206, 0);
font: 10pt &quot;幼圆&quot;;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="MyLabel" name="label">
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>开始游戏</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyLabel" name="label_2">
             <property name="text">
              <string>加载存档</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyLabel" name="label_3">
             <property name="text">
              <string>游戏设置</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyLabel" name="label_4">
             <property name="text">
              <string>退出游戏</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>label_3</zorder>
          <zorder>label_4</zorder>
          <zorder>label</zorder>
          <zorder>label_2</zorder>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="GamePage">
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>488</y>
          <width>1241</width>
          <height>182</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>160</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0" rowspan="2">
          <widget class="QLabel" name="label_charafacepic">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>160</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QLabel" name="label_charatalk">
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLabel" name="label_charaname">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_charaleft">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>480</width>
          <height>640</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_chararight">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>40</y>
          <width>480</width>
          <height>640</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>label_charaleft</zorder>
       <zorder>label_chararight</zorder>
       <zorder>frame</zorder>
      </widget>
      <widget class="QWidget" name="HistoryPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionMainPage"/>
    <addaction name="actionStart"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSetting"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuText">
    <property name="title">
     <string>文字</string>
    </property>
    <addaction name="actionHistory"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuText"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>开始游戏</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>加载存档</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>游戏设置</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出游戏</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>历史对话</string>
   </property>
  </action>
  <action name="actionMainPage">
   <property name="text">
    <string>回到标题</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
